layers/meta-resin-beaglebone/recipes-connectivity/bb-wl18xx-wlan0/bb-wl18xx-wlan0.bb

```diff
@@ -0,0 +1,21 @@
+DESCRIPTION = "Set wifi MAC address"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${RESIN_COREBASE}/COPYING.Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
+
+SRC_URI = " \
+	file://bb-wl18xx-wlan0 \
+	file://bb-wl18xx-wlan0.service \
+"
+
+inherit systemd
```

Comment:
We should inherit allarch too.
As well this should include a COMPATIBLE_MACHINE .